etc/org.opencastproject.kernel.security.LtiLaunchAuthenticationHandler.cfg

```diff
@@ -0,0 +1,42 @@
+# OAuth consumer keys with should be highly trusted.
+#
+# By default OAuth consumer are regarded as untrusted and a user authenticating via such
+# systems receives a rewritten username in the form of "lit:{ltiConsumerGUID}:{ltiUserID}".
+# This user is regarded as a new user temporarily existing for the duration of the session.
+# Opencast roles associated with the original user will not be attached to this user.
+#
+# Usernames of users authenticating via highly trusted systems will not be rewritten except
+# for the cases configured in the additional options below.
+#
+# Multiple consumer keys can be configured, by incrementing the counter. The list is read
+# sequentially incrementing the counter. If you miss any numbers it will stop looking for
+# further consumer keys.
+#lti.oauth.highly_trusted_consumer_key.1=CONSUMERKEY
+
+# Allow the Opencast system administrator user to authenticate as such via LTI.
+#
+# Note that this user may still authenticate via LTI, but the username will be rewritten,
+# even if a trusted OAuth consumer key is used.
+#
+# Note that this option does not apply to custom users having the ROLE_ADMIN. Use the
+# blacklist below instead.
+#
+# Default: false
+#lti.allow_system_administrator=true
+
+# Allow the Opencast digest user to authenticate as such via LTI.
+#
+# Note that this user may still authenticate via LTI, but the username will be rewritten,
+# even if a trusted OAuth consumer key is used.
+#
+# Default: false
+#lti.allow_digest_user=true
+
+# A blacklist of users not allowed to authenticate via LTI as themselves.
+#
+# Note that these users may still authenticate via LTI, but their username will be rewritten,
+# even if a trusted OAuth consumer key is used.
+#
+# Multiple users can be configured, by incrementing the counter. The list is read sequentially
+# incrementing the counter. If you miss any numbers it will stop looking for further users.
+#lti.blacklist.user.1=myAdminUser
```

Comment:
Should include the default (ie, no user is blacklisted).

Reply:
How would that look like? Just `#lti.blacklist.user.1=`?

Reply:
Yes, I think so.  And something like `Default: no blacklisted users`?